podcasts/Bookmarks/List/BookmarkRowViewModel.swift

```diff
@@ -0,0 +1,45 @@
+import SwiftUI
+import PocketCastsUtils
+import PocketCastsDataModel
+
+class BookmarkRowViewModel: ObservableObject {
+    let bookmark: Bookmark
+    private let imageManager: ImageManager
+
+    @Published var episodeImage: UIImage? = nil
+
+    let heading: String?
+    let title: String
+    let subtitle: String
+    let playButton: String
+
+    init(bookmark: Bookmark, imageManager: ImageManager = .sharedManager) {
+        self.imageManager = imageManager
+
+        self.bookmark = bookmark
+        self.title = bookmark.title
+        self.playButton = TimeFormatter.shared.playTimeFormat(time: bookmark.time)
+        self.subtitle = DateFormatter.localizedString(from: bookmark.created,
+                                                      dateStyle: .medium,
+                                                      timeStyle: .short)
+
+        guard let episode = bookmark.episode else {
+            self.heading = nil
+            return
+        }
+
+        _episodeImage = .init(initialValue: imageManager.placeHolderImage(.list))
+
+        self.heading = (episode as? Episode).flatMap {
+            $0.title
+        }
+
+        loadImage(for: episode)
+    }
+
+    private func loadImage(for episode: BaseEpisode) {
```

Comment:
[nit] While I believe this logic belongs here, I think we can use episode images in a lot of other places, so I wonder if a component specific for that — or an `Image` extension might be a good approach?

Reply:
That makes sense, I moved this into a separate component called `EpisodeImage`: dcded9238f3c94d7f0af386d6a84221dc66c461c